lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/ShowPageProducer.java

```diff
@@ -1261,7 +1261,15 @@ public void printSubpage(List<SimplePageItem> itemList, boolean first, UIBranchC
 			boolean forceButtonColor = false;
 			String color = null;
 			for (SimplePageItem i : itemList) {
-
+				
+				//If the content MULTIMEDIA, type 7, is not released or is hidden will not be rendered on the page
+				if(i.getType()==SimplePageItem.MULTIMEDIA) {
```

Comment:
Need some formatting here. Spaces after the if and spaces around the ==. It just makes it a bit easier to read.